tobac/segmentation.py

```diff
@@ -320,6 +320,8 @@ def segmentation_timestep(
     PBC_flag="none",
     seed_3D_flag="column",
     seed_3D_size=5,
+    below_threshold_num=0,
+    above_threshold_unseg_num=0,
```

Comment:
**Suggestion:** These parameter names are a bit unwieldy, and it may be difficult to change them after release. We could use a similar style to tracking which uses `cell_number_unassigned`, e.g. `segment_number_unassigned` and `segment_number_below_threshold`

Reply:
Great suggestion, the names I came up with were bad.